sass/styles/main.scss

```diff
@@ -412,6 +417,11 @@
     }
 }
 
+[dir='rtl'] .sc-input {
+    border-right: 1px solid $color-grey-5 !important;
```

Comment:
let's try getting rid of !important by finding a better place for this style.🙏🏻

Reply:
Actually In the actual .sc-input class there is property **border-right: none !important;**  . I don't know if we can overwrite that without important .